sequencer/sequencer/src/store/sled.rs

```diff
@@ -39,36 +44,65 @@ impl StoreEngine for Store {
             .map(|value| value.to_vec())
     }
 
-    fn add_transaction(&mut self, transaction_id: Key, transaction: Value) -> Result<()> {
-        let _ = self.transactions.insert(transaction_id, transaction);
-        Ok(())
+    fn add_transaction(&mut self, tx: Transaction) -> Result<()> {
```

Comment:
It would probably be cool to have a store API call to store multiple transactions in one go (for storing all transactions from a block at once) since these DBs support batches. Not for this PR but we should create an issue.